File: DTOs/Sources/RideServiceDTOs/PatchEventDTO.swift
import Foundation

public struct PatchEventDTO: Codable {
    public var name: String?
    public var description: String?
    public var starts: Date?
    public var ends: Date?
    public var latitude: Float?
    public var longitude: Float?
    
    public init(name: String? = nil, description: String? = nil, starts: Date? = nil, ends: Date? = nil, latitude: Float? = nil, longitude: Float? = nil) {
        self.name = name
        self.description = description
        self.starts = starts
        self.ends = ends
        self.latitude = latitude
        self.longitude = longitude
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/RideServiceDTOs/PatchEventDTO.kt
package net.ipv64.kivop.dtos.RideServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class PatchEventDTO (
    var name : String?,
    var description : String?,
    var starts : LocalDateTime?,
    var ends : LocalDateTime?,
    var latitude : Float?,
    var longitude : Float?,
)
